src/metabase/driver.clj

```diff
@@ -876,6 +876,10 @@
 ;;; |                                                    Upload                                                      |
 ;;; +----------------------------------------------------------------------------------------------------------------+
 
+(def ^:dynamic *insert-chunk-rows*
+  "The number of rows to insert at a time when uploading data to a database. This can be bound for testing purposes."
+  nil)
```

Comment:
why don't you set this to 100 by default?

Reply:
What's the benefit of that? I don't want it to look like there should be a common default value for all drivers. It's just a coincidence that it's 100 for all drivers at the moment.